src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { Tab } from '../Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleClick = (tab: Tab) => {
+    if (selectedTabId !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
```

Comment:
Use destructuring here.

Reply:
I can't understand how to use destructuring here to then use tab in onclick

Reply:
you could use this structure
```suggestion
      {tabs.map(tab => {
            const { id, title } = tab;
            return (
              your code here
            );
          })}
```